specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2022-02-14/imagebuilder.json

```diff
@@ -1226,6 +1226,7 @@
               "items": {
                 "$ref": "#/definitions/ImageTemplateInVMValidator"
               },
+              "x-ms-identifiers": [],
               "description": "List of validations to be performed."
             }
           }
```

Comment:
Have you tried building model for AIB? If these Swagger changes lead to any major changes in our model then there will be more work to fix the model. It would be nice to know what kind of change in the model this Swagger change will cause.

Reply:
I have not tried building the model for AIB yet. Once we get ARM and breaking change approval I will generate a new model.

Reply:
as of commit #11. i have copied the changes from this PR into our AIB and have no problems with datamodel (there are no changes to it)